pkg/contract_watcher/shared/parser/parser_test.go

```diff
@@ -220,7 +220,7 @@ var _ = Describe("Parser", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			selectMethods := p.GetMethods([]string{})
-			Expect(len(selectMethods)).To(Equal(22))
+			Expect(len(selectMethods)).To(Equal(25))
```

Comment:
@i-norden it seems like the changes to Geth caused this function to return more methods - do you know if this is a problem?

Reply:
It seems they fixed something.. inspecting the ABI for that contract shows that there are 25 functions, not 22... Not sure what exactly was changed such that 3 of the functions that were previously not considered methods are now considered as such, but I don't think it is a problem either way. Will update if I find anything else out!

Reply:
Just noticed this is a WIP, I may have approved prematurely 🙃